contrib/debian/README.md

```diff
@@ -15,7 +15,7 @@ To install:
 
 If you build yourself, you will either need to modify the paths in
 the .desktop file or copy or symlink your dash-qt binary to `/usr/bin`
-and the `../../share/pixmaps/dash128.png` to `/usr/share/pixmaps`
+and the `../../share/pixmaps/bitcoin128.png` to `/usr/share/pixmaps`
```

Comment:
Okok :) So we'll rename all bitcoin*.* files in share/pixmap and their references?

![image](https://cloud.githubusercontent.com/assets/636669/16490019/3add1aba-3ed8-11e6-835c-506dcfeba70b.png)